python/pyspark/sql/streaming/readwriter.py

```diff
@@ -1150,6 +1150,7 @@ def foreach(self, f: Union[Callable[[Row], None], "SupportsProcess"]) -> "DataSt
         This API is evolving.
 
         Examples
+        TODO(SPARK-43054): remove the SKIP flags below
```

Comment:
no biggie but should probably put this comment on the top of this method .. otherwise it'll be shown in API reference documentation ..